invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/ShareOptions/AccessUsers/UserAccessSearchResult.js

```diff
@@ -39,27 +40,65 @@ class UserAccessSearchResultItem extends Component {
     const { deleteSuccessCallback } = this.props;
     deleteSuccessCallback(data);
   };
+
+  accessDropdownOptions = [
+    {
+      key: "view",
+      text: i18next.t("Can view"),
+      value: "view",
+      description: i18next.t("Can view restricted records/files."),
+    },
+    {
+      key: "preview",
+      text: i18next.t("Can preview"),
+      value: "preview",
+      description: i18next.t("Can view drafts and restricted records/files."),
+    },
+    {
+      key: "edit",
+      text: i18next.t("Can edit"),
+      value: "edit",
+      description: i18next.t("Can edit drafts and view restricted records/files."),
+    },
+    {
+      key: "manage",
+      text: i18next.t("Can manage"),
+      value: "manage",
+      description: i18next.t(
+        "Can manage access, edit drafts and view restricted records/files."
+      ),
+    },
+  ];
+
+  dropdownOptionsGenerator = (value) => {
+    return value.map((options) => {
+      return {
+        key: options.key,
+        text: options.text,
+        value: options.key,
+        content: (
+          <>
+            {options.text}
+            <br />
```

Comment:
Could we use something else than `<br>`? Maybe some padding/margin or block elements?

Reply:
Used <div> for this. Does it look better?